tests/rootfs.sh

```diff
@@ -42,6 +48,9 @@ teardown()
         return
     fi
 
+    umount "${rootfs_dir}/${ARM_EMU_BIN}"
+    unlink "${rootfs_dir}/${ARM_EMU_BIN}"
```

Comment:
is rm not sufficient here?

Reply:
rm > unlink :p so technically everytime you do an rm, you should wonder, is unlink not sufficient here :)